src/main/java/teammates/common/util/Const.java

```diff
@@ -232,7 +228,6 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String COURSE_EMAIL_PENDING_COMMENTS = "Send email notification to recipients of %s pending comment%s";
         public static final String CLAIMED = "This is the student's own estimation of his/her contributions";
         public static final String PERCEIVED = "This is the average of what other team members think this student contributed";
-        public static final String PERCEIVED_CLAIMED = "Difference between claimed and perceived contribution points";
```

Comment:
Made obsolete in (but the constant was updated/renamed etc.)
~~~
commit 3ac4e8d25c4eb531857a7dc7fdea58917debe78c
Author: Damith Rajapakse <damith@gmail.com>
Date:   Sat Dec 1 10:35:43 2012 +0800

    Update Issue 403 [Remove 'unused-src' folder]
    
    
    Former-commit-id: 425bfb59da8e536a220e7ea7dda2bf99f59f1c1a
    Former-commit-id: e0e5f192cfe078f5cf8e15769b39ac676dc1c6bf
    Former-commit-id: 7a687b4f2dea238c4785ad2e59d567ab9c905e59
~~~